conduct.md

```diff
@@ -40,25 +40,34 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating.
+
+- Constructively criticize others.
+
+- Acknowledge others.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are discouraged to:
```

Comment:
Please refer to my other comments for these 3 sections. There should be adequate descriptions and/or examples of the kind of conduct that is outlined to avoid ambiguity.